src/file-picker/src/FilePicker.js

```diff
@@ -184,7 +188,20 @@ FilePicker.propTypes = {
    * Class name passed to the FilePicker.
    * Only use this if you know what you are doing.
    */
-  className: PropTypes.string
+  className: PropTypes.string,
+
+  /**
+   * Function that returns the call-to-action button text for selecting files.
+   * Gets the file count as number as parameter.
```

Comment:
```suggestion
   `* @type {(fileCount: number) => string}`
```

Reply:
sorry did not notice that there are ts typings